examples/resources/powerscale_snapshot_schedule/resource.tf

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright (c) 2023 Dell Inc., or its subsidiaries. All Rights Reserved.
+
+Licensed under the Mozilla Public License Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://mozilla.org/MPL/2.0/
+
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+# Available actions: Create, Update, Delete and Import
+# After `terraform apply` of this example file it will create NFS export on specified paths on the PowerScale Array.
+# For more information, Please check the terraform state file.
+
+# PowerScale SMB shares provide clients network access to file system resources on the cluster
+resource "powerscale_snapshot_schedule" "snap_schedule" {
+  # Required name of snapshot schedule for creating
+  name = "test_snap_schedule"
+
+  # Optional fields
+
+  # The path snapshotted. Default set to : "/ifs"
+  # path = "/ifs/tfacc_test"
+
+  # Time value in String for which snapshots created by this snapshot schedule should be retained. Values supported are of format : 
+  # "Never Expires, x Seconds(s), x Minute(s), x Hour(s), x Week(s), x Day(s), x Month(s), x Year(s) where x can be any integer value.
+  # Default set to : "1 Week(s)"
+  # retention_time = "3 Hour(s)" 
+
+  # Alias name to create for each snapshot.
+  # alias = "snap_alias"
```

Comment:
Does the Alias work correctly?

I noticed on snapshot even if you set the alias in the API it always just returns as `null` so I removed this field from snapshot.

Reply:
@doriac11 Yes it gives the value correctly if set.

Reply:
Okay thanks!